### PR TITLE
solana stablecoin balances

### PR DESCRIPTION
**Pull request by Synthquest**
## Thank you for contributing to Spellbook 🪄
Please open the PR in **draft** and mark as ready when you want to request a review. 

### Description:

Solana balances util table has gaps that exceed query capacity when trying to run. This fills null values on a subset of tokens

---
quick links for more information:
- [README.md](https://github.com/duneanalytics/spellbook/blob/main/README.md)
- [spellbook docs](https://github.com/duneanalytics/spellbook/tree/main/docs)
- [CONTRIBUTING.md](https://github.com/duneanalytics/spellbook/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by Synthquest** [comment]
cc: @jeff-dude

File: dbt_subprojects/solana/models/tokens/solana/tokens_solana_stablecoins.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{{ config(
+      schema = 'tokens_solana'
+      , alias = 'stablecoins'
+      , tags=['static']
+      , post_hook='{{ expose_spells(\'["solana"]\',
+                                  "sector",
+                                  "tokens_solana",
+                                  \'["synthquest"]\') }}'
+      , unique_key = ['address']
```
Comment (Hosuke) [inline dbt_subprojects/solana/models/tokens/solana/tokens_solana_stablecoins.sql:9]: unique_key is only required for incremental views.
```suggestion
```

File: dbt_subprojects/solana/models/tokens/solana/_schema.yml [hunk 2]
```diff
@@ -92,4 +92,27 @@ models:
...
+      tags: ['table', 'metadata', 'stablecoins', 'solana']
+    description: "Stablecoins on Solana blockchain with their details"
+    columns:
+      - name: blockchain
+        description: "The blockchain on which the stablecoin is issued"
+      - name: symbol
+        description: "The symbol of the stablecoin"
+      - name: address
+        description: "The address of the stablecoin on the Solana blockchain"
+      - name: decimals
```
Comment (Hosuke) [inline dbt_subprojects/solana/models/tokens/solana/_schema.yml:113]:
```suggestion
      - name: address
        description: "The address of the stablecoin on the Solana blockchain"
        data_tests:
          - unique
      - name: decimals
```

**Review by Hosuke — CHANGES_REQUESTED** [review]

**Comment by Synthquest** [comment]
Thanks @Hosuke 🙏

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**